src/Protocol/ActivityPub.php

```diff
@@ -113,7 +113,10 @@ private static function getAccountType($apcontact)
 			case 'Application':
 				$accounttype = User::ACCOUNT_TYPE_RELAY;
 				break;
-		}
+			case 'Tombstone':
+				$accounttype = User::ACCOUNT_TYPE_DELETED;
+				break;
+			}
```

Comment:
Please restore the original indentation.
```suggestion
		}
```

Reply:
Done